code/style.css

```diff
@@ -41,18 +41,69 @@ p {
   width: 250px;
   height: 400px;
   background: midnightblue;
+  text-align: center;
+  padding-top: 30px;
+  border-radius: 16px;
+}
 
+.mystery-deck {
+  width: 250px;
+  height: 400px;
+  background: rgb(112, 25, 86);
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
 }
 
 button {
   padding: 8px 16px;
-  background: crimson;
+  background: greenyellow;
```

Comment:
Smart to make the button green, in contrast to the red Reset-button! It's a detail, but it's really good that you thought about it. 👍